data-prepper-core/src/test/java/com/amazon/dataprepper/parser/ParseExceptionTest.java

```diff
@@ -0,0 +1,52 @@
+package com.amazon.dataprepper.parser;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.UUID;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+class ParseExceptionTest {
```

Comment:
This one might not be necessary since Data Prepper core does not require 100% test coverage.

However, I did improve the original PipelineParserTests by updating names and exception handling. While working it, I found [this test](https://github.com/opensearch-project/data-prepper/blob/main/data-prepper-core/src/test/java/com/amazon/dataprepper/parser/PipelineParserTests.java#L128-L143). It appears to be trying to test the exception, just in another class. I moved it here rather than dropping it entirely.